V1_RELEASE_NOTES.md

```diff
@@ -3,3 +3,46 @@
 ## Deprecated
 
 * `.touch-border` has been removed as it's was considered too specific.
+
+### Settings changes
+
+
+* `$mobile-nav` has been removed
+* `$nav-active-bg` has been removed
+* `$bullet-color` has been removed
+* `$primary-button-color` has been removed
+* `$box-border` has been removed
+* `$table-th` has been removed
+* `$code-bg` has been removed
+* `$box-shadow` has been removed
+* `$box-solid-grey` has been removed
+* `$information` has been removed
+* `$row-background` has been removed
+* `$header-link-color` has been removed
+* `$text-color` has been removed
+* `$transparent` is now `$color-transparent`
+* `$brand-color` is now `$color-brand`
+* `$brand-color-light` is now `$color-brand-light`
+* `$cool-grey` is now `$color-cool-grey`
+* `$warm-grey` is now `$color-warm-grey`
+* `$mid-grey` is now `$color-mid-grey`
+* `$light-grey` is now `$color-light-grey`
+* `$alto-grey` is now `$color-alto-grey`
+* `$white` is now `$color-white`
+* `$black` is now `$color-black`
+* `$error` is now `$color-error`
+* `$warning` is now `$color-warning`
+* `$success` is now `$color-success`
+* `$link-color` is now `$color-highlight`
+
+* `$gutter-width` is now `$grid-gutter-width`
+* `$columns` is now `$grid-columns`
+* `$site-max-width` is now `$grid-max-width`
+
+* `$navigation-threshold` has been replaced with `$breakpoint-small`
+
+* `$asset-server` is now `$assets-server`
```

Comment:
I would like to rename this one too `$assets-path` as it is used to link to local assets when the site news to work offline.